### PR TITLE
Command line configuration file

### LINKED ISSUE #62: Enable loading all configuration options from a file.
Currently you can load options to do with queries from a file, but not all the command line options. I'd like to be able to specify those in a configuration file, so that file can be kept in a separate place. I'm planning to work on this issue, probably just by adding this https://pypi.org/project/click-config-file/ and testing it a bit.

### PR DESCRIPTION
**Pull request by RobertKirk**
Using `click-config-file`, we can easily load all command line options from a file.

Also adding documentation for developing inside the docker container. This is another way to develop apart from creating a python virtualenv, or just installing into the system environment.

Closes #62

### CODE REVIEW

File: prometheus_es_exporter/__init__.py [hunk 1]
```diff
@@ -392,9 +393,11 @@ def conv(value):
               help='Detail level to log. (default: INFO)')
 @click.option('--verbose', '-v', default=False, is_flag=True,
               help='Turn on verbose (DEBUG) logging. Overrides --log-level.')
+@click_config_file.configuration_option()
 def cli(**options):
     """Export Elasticsearch query results to Prometheus."""
-
+    print(options)
+    return
```
Comment (braedon) [inline prometheus_es_exporter/__init__.py:400]: I presume this is left over from testing?

File: README.md [hunk 2]
```diff
@@ -83,4 +82,18 @@ To build a docker image directly from the git repo, run the following in the roo
 ```
 > sudo docker build -t <your repository name and tag> .
 ```
+
+To develop in a docker container file you should first build the container using
+```
+> sudo docker build -t exporter:latest
+```
+Then run (in the root of the git repo)
```
Comment (braedon) [inline README.md:90]:
```suggestion
To develop in a docker container, first build the image, and then run the following in the root project directory:
```

File: README.md [hunk 3]
```diff
@@ -83,4 +82,18 @@ To build a docker image directly from the git repo, run the following in the roo
...
+```
+> sudo docker run --rm -it --name exporter --entrypoint bash $(pwd):/usr/src/app exporter:latest
```
Comment (braedon) [inline README.md:92]:
```suggestion
> sudo docker run --rm -it --name exporter --entrypoint bash -v $(pwd):/usr/src/app <your repository name and tag>
```

File: prometheus_es_exporter/__init__.py (hunk 1 above)
Comment (RobertKirk) [reply prometheus_es_exporter/__init__.py]: Yeah. I've removed it

**Comment by RobertKirk** [comment]
Thanks for the comments, have fixed and accepted your changes. It seems the branch got screwed up a bit, hence the force-pushes, but will squash the commits when you approve, or can do it now?

**Comment by braedon** [comment]
No worries, I'll do a squash when I merge.

**Comment by RobertKirk** [comment]
Great. Thanks for the fast responses :smile:

**Comment by braedon** [comment]
(I'll release v0.9.0 in the next few days)

**Comment by RobertKirk** [comment]
Great, thanks

**Comment by braedon** [comment]
Released in [0.9.0](https://github.com/braedon/prometheus-es-exporter/releases/tag/0.9.0)

**Pull request merged**